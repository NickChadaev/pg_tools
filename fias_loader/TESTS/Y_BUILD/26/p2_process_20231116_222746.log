2023-11-16 22:27:46 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_26 postgres /home/n.chadaev@abrr.local/Y_BUILD/26/stage_gar_c_26.csv 2023-11-13
--------------------------------------------------------
[22:27:46] ... Ставропольский край, id_region: 27, Execute: /home/n.chadaev@abrr.local/Y_BUILD/26/stage_gar_c_26.csv, process_id:25345
[22:27:46] ... Start process
[22:27:46] ...  -- Version
[22:27:46] ...  -- Очистка от данных
[22:27:46] ...  -- Убираем индексы
[22:27:46] ...  -- Set UNLOGGED
[22:27:46] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[22:27:47] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[22:27:47] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[22:27:47] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[22:27:47] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[22:27:47] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[22:27:47] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[22:27:47] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[22:27:47] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[22:27:47] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[22:27:47] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_REESTR_OBJECTS_20231113_fda41729-9152-468d-aa63-2cea03d1069d.XML
[22:56:44] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_ADDR_OBJ_20231113_41122228-3929-4f0b-ba39-1c71662bf895.XML
[22:57:11] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_HOUSES_20231113_7f0479ce-ce77-4742-81c7-e3f7d2cd1873.XML
[23:21:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_STEADS_20231113_2de706d2-244d-4a60-8777-d4d0597ecc84.XML
[23:31:48] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_APARTMENTS_20231113_9cc38f34-528f-4305-bc19-5df62e092c53.XML
[23:40:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_ROOMS_20231113_7595e767-6a8c-4363-a7f6-f86816a32680.XML
[23:40:21] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_CARPLACES_20231113_51058856-5dae-4fc1-907e-6c96de7c711a.XML
[23:40:24] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_ADM_HIERARCHY_20231113_cc6a56f6-ca8e-4a53-b27a-d10492ad9037.XML
[00:12:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_MUN_HIERARCHY_20231113_e772be66-7ab3-443d-b6d2-adaf02247324.XML
[00:50:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_CHANGE_HISTORY_20231113_5d0ca090-dd47-41fc-91e5-97ac7d30cff1.XML
[02:49:04] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_ADDR_OBJ_DIVISION_20231113_851b455d-7440-46ff-8b47-485fc918296a.XML
[02:49:04] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_ADDR_OBJ_PARAMS_20231113_98ebe820-b28e-4c4c-852f-61199144f3df.XML
[02:53:53] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_STEADS_PARAMS_20231113_7cffd6a6-ab70-4d9c-a8c9-8ecfb8080e25.XML
[05:05:17] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_HOUSES_PARAMS_20231113_60603378-c463-4401-b7b4-a245f45d938f.XML
[07:34:52] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_APARTMENTS_PARAMS_20231113_36920bfe-d1c4-47b3-9dfd-bba02436ee42.XML
[07:50:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_ROOMS_PARAMS_20231113_12b24bc4-2e9d-462f-baa4-db4ed2fcf9c9.XML
[07:50:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_CARPLACES_PARAMS_20231113_d7f9e53a-bb62-444e-9727-e01ad08f4d39.XML
[07:50:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/26/AS_NORMATIVE_DOCS_20231113_24350263-1c83-40b7-9216-057cdabee7ae.XML
[07:54:01] ...  -- Data finish
[07:54:01] ...  -- Set LOGGED
[07:55:20] ... Stop process
--------------------------------------------------------
