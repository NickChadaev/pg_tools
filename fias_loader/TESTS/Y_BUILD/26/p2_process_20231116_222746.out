 save_gar_version 
------------------
               81
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                81
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 2342831
gar_fias.as_addr_obj: 30783
gar_fias.as_houses: 1796067
gar_fias.as_steads: 841307
gar_fias.as_apartments: 682731
gar_fias.as_rooms: 2118
gar_fias.as_carplaces: 3649
gar_fias.as_adm_hierarchy: 2501796
gar_fias.as_mun_hierarchy: 3073301
gar_fias.as_change_history: 9877597
gar_fias.as_addr_obj_division: 111
gar_fias.as_addr_obj_params: 385283
gar_fias.as_steads_params: 10782410
gar_fias.as_houses_params: 12116435
gar_fias.as_apartments_params: 1276349
gar_fias.as_rooms_params: 2401
gar_fias.as_carplaces_params: 234
gar_fias.as_normative_docs: 282322
